README.md

```diff
@@ -7,16 +7,54 @@ See [wasix.org](https://wasix.org) for more.
 
 ## Installation
 
-To install this Cargo subcommand, first you'll want to [install
-Rust](https://www.rust-lang.org/tools/install) and then you'll execute:
+You can install this Cargo subcommand via:
 
-```
+> Note: Installation requires [Rust](<(https://www.rust-lang.org/tools/install)>) be installed on your system.
+
+### Information
+
+This subcommand is available on [crates.io](https://crates.io/crates/cargo-wasix)
+
+Available for platforms:
+
+- `x86_64-unknown-linux-gnu`
+- `x86_64-apple-darwin`
+- `x86_64-apple-darwin`
+- `x86_64-pc-windows-msvc`
+
+### Cargo Install
+
+```shell
 $ cargo install cargo-wasix
 ```
 
-After that you can verify it works via:
+### Cargo Binstall
+
+> Uses pre-built binaries.
```

Comment:
Maybe link to bininstall and a quick explanation what it is?

Reply:
added explaination